pkgs/tools/misc/picocom/default.nix

```diff
@@ -18,6 +19,15 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-cs2bxqZfTbnY5d+VJ257C5hssaFvYup3tBKz68ROnAo=";
   };
 
+  patches = [
+    # support high baud rates on Apple arm64 processor
+    # https://github.com/npat-efault/picocom/pull/129
+    (fetchpatch {
+      url = "https://github.com/npat-efault/picocom/commit/f806bf28266cccdb75ba89d754de8d8fa64c6127.patch";
+      hash = "sha256-shAJHr0sgzGu7LInP1d9NXPA1A8uXr0RUQOgsH6JYQw=";
+    })
```

Comment:
If we want to stick with patching instead of moving to the active fork, we should vendor this in the repository (see https://github.com/NixOS/nixpkgs/pull/341131#discussion_r1753046220 for rationale).

Reply:
I was also sceptical since I’d never seen it happen until https://github.com/NixOS/nixpkgs/pull/123456 was arbitrarily pulled out of a hat for me – you can see the old commits there are 404s now. For long‐dormant PRs it’s admittedly not a significant risk, so I probably over‐bikeshedded here, sorry. (And FODs in general are a big risk anyway – probably we need some automated checking for things like this, though Nix being dynamic makes it hard. I’ve had thoughts about a `fetchGitHubPR` with a caching MITM that could paper over the problems here.)

Reply:
> I was also sceptical since I’d never seen it happen until #123456 was arbitrarily pulled out of a hat for me – you can see the old commits there are 404s now. 

hmm, i can't seem to find the 404s or patches used in that PR but i will take your word that it can happen.

> For long‐dormant PRs it’s admittedly not a significant risk, so I probably over‐bikeshedded here, sorry. 

if it can happen seems legit request. will vendor from now on. thanks!